frontend/src/components/GoogleMaps/GooglePlacesInput.tsx

```diff
@@ -77,7 +93,9 @@ const GooglePlacesInput = ({ onLocationSelect, value }: GooglePlacesInputProps)
         value={radius}
         onValueChange={handleSliderChange}
       />
-      <Text className='font-semibold text-base'>Current radius: {(radius * 0.000621371).toFixed(2)} miles</Text>
+      <Text className="font-semibold text-base">
+        Current radius: {(radius * 0.000621371).toFixed(2)} miles
```

Comment:
We shouldn't have magic number floating around. It makes the code harder to maintain.